sdcm/cluster.py

```diff
@@ -2887,6 +2890,7 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
         self.params = params
         self.datacenter = region_names or []
         self.dead_nodes_ip_address_list = set()
+        self.use_saslauthd_authenticator = self.params.get('use_saslauthd_authenticator')
```

Comment:
Where is "use_saslauthd_authenticator" set?

Reply:
For using com.scylladb.auth.SaslauthdAuthenticator, saslauthd process should be started, it's a broker between scylla and ldap server.

Reply:
But i don't see this parameter in any yaml.

Reply:
> But i don't see this parameter in any yaml.

It will only be enabled for Scylla Enterprise, it's not supported by Scylla master.

Reply:
I did a wrong description.

use_saslauthd_authenticator parameter will only effect get_db_auth() for workloads or nodetool cmd.

In the new nemesis, we don't need to enable it in the yaml config. The switch occurred during the nemesis.

But configurations/ldap-authenticator.yaml and configurations/ldap-authorization-and-authentication.yaml are used to always enable SaslauthdAuthenticator for the whole job, so I just add `use_saslauthd_authenticator: true` in them.